src/fragments/start/getting-started/next/api.mdx

```diff
@@ -346,7 +360,10 @@ Submit that form to create a new post, and you'll build that page next!
 
 Statically generating pages during the build process improves performance. But, dynamically created posts still need to not `404`.
 
-To solve this, create **pages/posts/[id].js** and paste in the following content:
+To solve this, create a new folder within **/pages** named "posts".  Next, create a new file within this new **/posts** named **[id].js** to create the structure: 
```

Comment:
this change seems unnecessary to me but if it slowed you down at all while you were running through the guide go ahead with it

Reply:
That's fair feedback.  I felt it was a little verbose too.  I'll rework it to something shorter and/or just revert to original.